commands/http/handler.go

```diff
@@ -182,6 +174,14 @@ func sendResponse(w http.ResponseWriter, req cmds.Request, res cmds.Response) {
 		h.Set(contentLengthHeader, strconv.FormatUint(res.Length(), 10))
 	}
 
+	if _, ok := res.Output().(io.Reader); ok {
+		// we don't set the Content-Type for streams, so that browsers can MIME-sniff the type themselves
+		// we set this header so clients have a way to know this is an output stream
+		// (not marshalled command output)
+		mime = ""
+		h.Set(streamHeader, "1")
```

Comment:
streamheader means that youre getting an arbitrary stream of data. `chan interface{}` means youre getting a series of json objects.